lib/MetaCPAN/Model/Search.pm

```diff
@@ -44,6 +44,16 @@ sub search_simple {
     return $results;
 }
 
+sub search_first {
```

Comment:
i'll change the name as suggested.
just to be clear, it's the only usage of search_simple... so technically can be used only as a temp. name until we remove the need for search_simple